src/exportstep.cpp

```diff
@@ -5,6 +5,235 @@
 //-----------------------------------------------------------------------------
 #include "solvespace.h"
 
+const double PRECISION = 0.001;
```

Comment:
```suggestion
const double PRECISION = 2*LENGTH_EPS;
```

Reply:
Defined [here](https://github.com/solvespace/solvespace/blob/master/src/solvespace.h#L129) used everywhere as a threshold for numerical solutions. `2*LENGTH_EPS` should guarantee (I would think) matching points.

0.001 is too big - there are people (very rarely) doing (for whatever reason) very precise things.